test/sparse/sparse.jl

```diff
@@ -1619,6 +1619,10 @@ end
     @test norm(Ai,1) ≈ norm(Array(Ai),1)
     @test norm(Ai,Inf) ≈ norm(Array(Ai),Inf)
     @test vecnorm(Ai) ≈ vecnorm(Array(Ai))
+    # make certain entries in nzval beyond
+    # the range specified in colptr do not
+    # impact vecnorm of a sparse matrix
+    @test (foo = speye(4); resize!(foo.nzval, 5)[5] = NaN; vecnorm(foo) == 2.0)
```

Comment:
Please use multiple lines

Reply:
Cheers, I will extend this test to multiple lines and then merge. Thanks!